src/components/OrgPostCard/components/PostEditModal.tsx

```
@@ -0,0 +1,284 @@
+    <Modal
+      show={show}
+      onHide={onHide}
+      backdrop="static"
+      centered
+      backdropClassName="bg-dark"
+    >
+      <Modal.Header closeButton className={styles.modalHeader}>
+        <Modal.Title>{t('editPost')}</Modal.Title>
+      </Modal.Header>
+
+      <Form onSubmit={updatePost} role="form" data-testid="update-post-form">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<form>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)